src/tensors/cpu/prod.cpp

```diff
@@ -195,9 +197,10 @@ void ProdBatched(marian::Tensor C,
                  bool transB,
                  float beta,
                  float scalar) {
-  if (C->getBackend()->isLegacyBatchedGemm()) {
+  //if (C->getBackend()->isLegacyBatchedGemm()) {
     ProdBatchedOld(C, allocator, A, B, transA, transB, beta, scalar);
-  }
+  //}
+  /*
```

Comment:
... and below: Clean up please before submitting pull requests.

Reply:
I am fine with cleaning up this file. However, I would like to have @XapaJIaMnu's opinion.
@XapaJIaMnu As I did this change [taking your code as reference](https://github.com/browsermt/marian-dev/pull/8), I would like to know if I can remove the dead code from here 👍

Reply:
What I did here is an example to show you how to remove MKL from the codebase (and replace it with oneDNN). Since then there were other changes on top of my example (Did we end up using Eigen as a BLAS library?). You can safely remove the commented out lines, as there is no `sgemmBatched` implementation with the BLAS provider.

Reply:
@XapaJIaMnu Just to be sure, I can remove [this dead code](https://github.com/browsermt/marian-dev/blob/wasm/src/tensors/cpu/prod.cpp#L203-L280) as well?

Reply:
Cool. As @XapaJIaMnu confirmed in the comments, I will clean this up.